### ISSUE TITLE
WebRTC IP per service

### ISSUE THREAD

**Issue by Devorlon** [ferdium/ferdium-app#1398]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/HEAD/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22enhancement+%3Asparkles%3A%22) for a feature request that matches the one I want to file, without success.

### Problem Description

To have discord and other real time communication services work you need to have WebRTC enabled, but for other services like email and random websites it's not necessary.

### Proposed Solution

An option in the settings of each service to set the WebRTC policy, the global option should still be available just the individual setting would override it.

**Comment by ChaoticHolon** [comment]
Yes, this is very necessary for some web apps like Discord and Google Voice that require at bare minimum that the WebRTC public IP is exposed for real time communication functionality to work (it's possible that local IP may need to be exposed for some apps as well). Most apps **_do not need this_**, but some of them definitely do, so it would be very helpful to have this as a per service setting INSTEAD of just a global one. 

The present setup requires us to either turn it on/off globally and restart Ferdium every time this is toggled (I assume most users won't bother doing it this way), or just leave WebRTC IP exposure on all the time, effectively sacrificing security/privacy for convenience (as doing that exposes IPs in every service globally, all the time).

Ideally we should only need to toggle that setting per service that needs those IPs exposed for full functionality, and if necessary, just reload the service instead of having to restart Ferdium as a whole.
